benchmarks/benchmarks/bench_core.py

```diff
@@ -48,6 +50,12 @@ def time_array_l_view(self):
     def time_can_cast(self):
         np.can_cast(self.l10x10, self.float64_dtype)
 
+    def time_tobytes_noncontiguous(self):
+        self.noncontig.tobytes()
+
+    def time_tobytes_contiguous(self):
+        np.ascontiguousarray(self.noncontig).tobytes()
```

Comment:
I don't think we should time this.  You could use `.T` just directly in the non-contiguous version also (since the `.T` will be irrelevantly fast here).